e2e_deployment_files/long_haul_deployment_windows.template.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
Please add comments to mention issues about snitcher and influxdb.

Nit: suggest to rename this file to "long_haul_deployment.template.windows.json"